sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -216,14 +238,18 @@ abstract class ScrollableState<T extends Scrollable> extends State<T> {
     _animation.stop();
   }
 
+  void _handleDragStart() {
```

Comment:
why does this method exist? Can't the callers just invoke dispatchOnScrollStart directly?